src/toTransform.js

```diff
@@ -8,38 +8,78 @@
     return !isNaN(number);
   }
 
-  function convertTranslate (values) {
-    if (values === undefined || values === 'none') {
+  function convertTranslate (input) {
+    if (input === undefined || input === 'none') {
       return null;
     }
 
-    var valuesArray = values.split(' ');
-    var numValues = valuesArray.length;
+    var values = input.split(' ');
+    var numValues = values.length;
 
     if (numValues > 3) {
       throw new InvalidArgument('Too many values for translate');
     }
 
     for (var i = 0; i < numValues; i++) {
-      var elementLength = valuesArray[i].length;
+      var elementLength = values[i].length;
       if (elementLength <= 2) {
         throw new InvalidArgument('Incorrect argument for translate');
       }
 
       // Check if units are px
-      var unit = valuesArray[i].substring(elementLength - 2, elementLength);
+      var unit = values[i].substring(elementLength - 2, elementLength);
       if (unit !== 'px') {
         throw new InvalidArgument('Incorrect units for translate');
       }
 
       // Check if characters before 'px' are valid numbers
-      var number = valuesArray[i].substring(0, elementLength - 2);
+      var number = values[i].substring(0, elementLength - 2);
       if (!isNumeric(number)) {
         throw new InvalidArgument('Argument must be a number');
       }
     }
 
-    return 'translate(' + valuesArray.join(', ') + ')';
+    return 'translate(' + values.join(', ') + ')';
+  }
+
+  function convertRotate (input) {
+    if (input === undefined || input === 'none') {
+      return null;
+    }
+
+    var values = input.split(' ');
+    var numValues = values.length;
+
+    if (numValues !== 4 && numValues !== 1) {
+      throw new InvalidArgument('Incorrect number of arguments for rotate');
+    }
+
+    var angle = values[numValues - 1];
+    var angleLength = angle.length;
+
+    var angleUnitArray = /deg|grad|rad|turn$/.exec(angle);
```

Comment:
I think the |s will need to be in ()s for the $ to apply to all of them. At the very least it will make it clear to the reader that's what you intended.